setup_dev_from_wheel.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+# Python environment name
+PYTHON_ENV_NAME=$1
+
+# Full CPL suite Wheels or Catalyst only
+# full | catalyst-only
+PACKAGE_WHEELS=$2
+
+# Exit on any error
+set -e
+
+# Turn off detached head advice on Git
+git config --local advice.detachedHead false
+
+export CATALYST_DIR=$PWD
+export CATALYST_FRONTEND_SRC=$CATALYST_DIR/frontend/catalyst
+export CATALYST_PYTHON_ENV_DIR=/tmp/$PYTHON_ENV_NAME
+
+install_wheels(){
+    # Create (if not created yet) and activate the virtual environment 
+    python3 -m venv $CATALYST_PYTHON_ENV_DIR
+    source $CATALYST_PYTHON_ENV_DIR/bin/activate
+    echo "Success: The Python virtual environment located at '$CATALYST_PYTHON_ENV_DIR' was activated"
+
+    # Clean the Catalyst installation
+    python -m pip uninstall -y pennylane-catalyst
+
+    # Choose between a full CPL install or a Catalyst only
+    if [ "$PACKAGE_WHEELS" = "catalyst-only" ]; then
+        echo "Installing Wheels from Catalyst only..."
+        python -m pip install --extra-index-url https://test.pypi.org/simple/ \
+        pennylane-catalyst --pre --upgrade
+    elif [ "$PACKAGE_WHEELS" = "full" ]; then
+        echo "Installing full CPL Wheels..."
+        python -m pip install --extra-index-url https://test.pypi.org/simple/ \
+        pennylane pennylane-lightning pennylane-catalyst --pre --upgrade
```

Comment:
I don't think there is a distinction between these two anymore (catalyst depends on PL & Lightning)

Reply:
Removed. Done.